From da3a601d488f3ba6a61baee4b4e4545f21d238ab Mon Sep 17 00:00:00 2001
From: Simon Glass <sjg@chromium.org>
Date: Sat, 19 Aug 2017 11:09:02 -0600
Subject: [PATCH 26/29] README: Add a note about test_tree1.dts

Add a little note in the README about something which confused me.

Signed-off-by: Simon Glass <sjg@chromium.org>
---
 README | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/README b/README
index 17dc845..15232ab 100644
--- a/README
+++ b/README
@@ -73,6 +73,17 @@ More work remains to support all of libfdt, including access to numeric
 values.
 
 
+Tests
+-----
+
+Test files are kept in the tests/ directory. Use 'make check' to build and run
+all tests.
+
+If you want to adjust a test file, be aware that tree_tree1.dts is compiled
+and checked against a binary tree from assembler macros in trees.S. So
+if you change that file you must change tree.S also.
+
+
 Mailing list
 ------------
 The following list is for discussion about dtc and libfdt implementation
-- 
2.14.1.480.gb18f417b89-goog

